storage/pathmanager/pathManager.go

```diff
@@ -0,0 +1,69 @@
+package pathmanager
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/ElrondNetwork/elrond-go/storage"
+)
+
+const (
+	shardPlaceholder      = "[S]"
```

Comment:
If you make these consts exported, you can use them inside main.go when defining the path templates.

Reply:
Yes, or like this, if they are not defined in the "core" package and be used everywhere.

Reply:
done. moved them to core->constants